src/diffusers/pipelines/ddpm/pipeline_ddpm.py

```diff
@@ -73,7 +73,7 @@ def __call__(
             "Please make sure to instantiate your scheduler with `prediction_type` instead. E.g. `scheduler ="
             " DDPMScheduler.from_pretrained(<model_id>, prediction_type='epsilon')`."
         )
-        predict_epsilon = deprecate("predict_epsilon", "0.10.0", message, take_from=kwargs)
```

Comment:
The `predict_epsilon` deprecation was only introduced 3 weeks ago, and we've had two rapid releases since then, so I'm keen on keeping it at least until next week.

Reply:
Agree - thanks for double-checking!